internal/controller/thanosquery_controller_test.go

```diff
@@ -101,14 +95,6 @@ var _ = Describe("ThanosQuery Controller", Ordered, func() {
 			}
 			By("setting up the thanos query resources", func() {
 				Expect(k8sClient.Create(context.Background(), resource)).Should(Succeed())
-				reg := prometheus.NewRegistry()
```

Comment:
Thanks for removing it, this makes sense, but I keep seeing test locally fail when I remove them. Maybe envtest issue?